CHANGES.rst

```diff
@@ -10,6 +10,8 @@ Changelog
 
 - Add support for Python 3.7.
 
+- Avoid RuntimeError in _takeOutGarbage. See `issue 7
+  <https://github.com/zopefoundation/tempstorage/issues/7>`_.
 
```

Comment:
Minor: before your change there were two blank lines above the `4.0.1` section title, now only one is left.

(I personally prefer two blank lines before each ReST header.)